pymess/models/push.py

```diff
@@ -71,7 +73,9 @@ def get_controller(self):
     def send(self, recipient, context_data, related_objects=None, tag=None, **kwargs):
         return super().send(recipient, context_data, related_objects, tag,
                             state=AbstractPushNotificationMessage.STATE.WAITING,
-                            heading=self.render_text_template(self.heading, context_data, recipient), **kwargs)
+                            heading=self.render_text_template(self.heading, context_data, recipient),
+                            redirect_url=self.render_text_template(self.redirect_url, context_data, recipient),
```

Comment:
As `redirect_url` is nullable, you have to ensure it's not `None` before rendering. Otherwise, django template engine will render it literally as `None`.

Reply:
oh, that's right. Fixed. Thanks